src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -1067,6 +1078,607 @@ TEST_F(MessageHelperTest,
+
+  MessageHelper::SendQueryApps(kConnectionKey, mock_application_manager);
+
+  ASSERT_TRUE(result.valid());
+
+  EXPECT_EQ(mobile_apis::FunctionID::OnSystemRequestID,
+            (*result)[strings::params][strings::function_id].asInt());
+}
+
+TEST_F(MessageHelperTest, SendPolicyUpdate_SUCCESS) {
+  const std::string kFilePath("file_path");
+  const int kTimeout = 0u;
+  std::vector<int> dummy_retries;
+  dummy_retries.push_back(5);
+
+  smart_objects::SmartObjectSPtr result;
+  EXPECT_CALL(mock_application_manager, ManageHMICommand(_))
+      .WillOnce(DoAll(SaveArg<0>(&result), Return(true)));
+
+  MessageHelper::SendPolicyUpdate(
+      kFilePath, kTimeout, dummy_retries, mock_application_manager);
+
+  ASSERT_TRUE(result.valid());
+
+  EXPECT_EQ(hmi_apis::FunctionID::BasicCommunication_PolicyUpdate,
+            (*result)[strings::params][strings::function_id].asInt());
+}
+
+TEST_F(MessageHelperTest, SendSystemRequestNotification_SUCCESS) {
+  const uint32_t kConnectionKey = 1u;
+
+  smart_objects::SmartObject result;
+  EXPECT_CALL(mock_application_manager, ManageMobileCommand(_, _))
+      .WillOnce(Return(true));
+
+  MessageHelper::SendSystemRequestNotification(
+      kConnectionKey, result, mock_application_manager);
+
+  EXPECT_EQ(mobile_apis::FunctionID::OnSystemRequestID,
+            result[strings::params][strings::function_id].asInt());
+}
+
+TEST_F(MessageHelperTest, SendTTSGlobalProperties_SUCCESS) {
+  const bool kDefaultHelpPrompt = true;
+  MockApplicationSharedPtr app(::utils::MakeShared<MockApplication>());
+
+  CommandsMap commands_map;
+
+  smart_objects::SmartObject command(smart_objects::SmartType_Map);
+  command[strings::menu_params][strings::menu_name] = "dummy_menu_name";
+
+  commands_map.insert(CommandsMap::value_type(0u, &command));
+
+  ON_CALL(*app, commands_map())
+      .WillByDefault(Return(DataAccessor<CommandsMap>(commands_map, lock_)));
+
+  smart_objects::SmartObjectSPtr result;
+  EXPECT_CALL(mock_application_manager, ManageHMICommand(_))
+      .WillOnce(DoAll(SaveArg<0>(&result), Return(true)));
+
+  MessageHelper::SendTTSGlobalProperties(
+      app, kDefaultHelpPrompt, mock_application_manager);
+
+  ASSERT_TRUE(result.valid());
+
+  EXPECT_EQ(hmi_apis::FunctionID::TTS_SetGlobalProperties,
+            (*result)[strings::params][strings::function_id].asInt());
+}
+
+TEST_F(MessageHelperTest, SendShowRequestToHMI_NoApp_UNSUCCESS) {
+  EXPECT_CALL(mock_application_manager, ManageHMICommand(_)).Times(0);
+
+  ApplicationSharedPtr dummy_app;
+  MessageHelper::SendShowRequestToHMI(dummy_app, mock_application_manager);
+}
+
+TEST_F(MessageHelperTest, SendShowRequestToHMI_SUCCESS) {
+  MockApplicationSharedPtr app(::utils::MakeShared<MockApplication>());
+
+  smart_objects::SmartObject dummy_show_command(smart_objects::SmartType_Map);
+
+  ON_CALL(*app, show_command()).WillByDefault(Return(&dummy_show_command));
+
+  smart_objects::SmartObjectSPtr result;
```

Comment:
@LevchenkoS , i see this "result" in every test. Can you move it to test fixture?